infrastructure/users/users.go

```diff
@@ -30,7 +30,7 @@ type Repository struct {
 
 func (r *Repository) GetByUsername(name string) (*model.User, error) {
 	var user model.User
-	result := r.db.Where("username = ?", name).
+	result := r.db.Preload("Followers").Where("username = ?", name).
```

Comment:
We use GetByUsername in a few places, we might want a separate method to get the user with followers